doc/man/8/ceph-volume.rst

```diff
@@ -15,7 +15,6 @@ Synopsis
 **ceph-volume** **inventory**
 
 **ceph-volume** **lvm** [ *trigger* | *create* | *activate* | *prepare*
-| *zap* | *list* | *batch*]
```

Comment:
IIUC, deleting this line was by accident?

Reply:
yep, fixed.

Reply:
Thanks!